creator-cards/owain-williams.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Owain Williams",
+  "sectionTitle": null,
+  "countryCode": "uk",
+  "slogan": "",
+  "type": "CREATOR",
+  "bio": "Hi I'm Owain, I'm an Umbraco CMS MVP, .Net Meetup organiser and Lego fan",
+  "tags": ".NET, C#, ASP.NET Core, Umbraco CMS",
+  "socials": {
+    "youtube": "",
+    "linkedin": "https://www.linkedin.com/in/owainwilliams//",
+    "twitter": "https://twitter.com/owaincodes",
+    "github": "https://github.com/OwainWilliams",
+    "mastodon": "",
+    "sessionize": ""
+  },
+  "channels": [
+    {
+      "name": "Blog",
+      "url": "https://owain.codes"
+    }
+  ],
+  "feeds": [
+    {
+      "type": "LinkTagRssType",
```

Comment:
Hey @OwainWilliams we changed this a while ago to be either "dotnet" or "personal"

Dotnet is used if your feed ONLY contains  .NET, azure etc related content
and personal if the feed contains anything else like social stuff, personal posts etc etc

If you have both in your feed, can you give us a filtered feed for dotnet and a personal one?

if you only have related content than just rename the type to "dotntet"

Reply:
Thanks @DeeJayTC  - updated. :)